frontend/src/metabase/browse/components/BrowseApp.tsx

```diff
@@ -55,27 +55,29 @@ export const BrowseApp = ({
         <BrowseDataHeader>
           <BrowseSectionContainer>
             <h2>{t`Browse data`}</h2>
-            <div
-              className="flex flex-align-right"
-              style={{ flexBasis: "40.0%" }}
-            >
-              <Link className="flex flex-align-right" to="reference">
-                <BrowseHeaderIconContainer>
-                  <Icon
-                    className="flex align-center"
-                    size={14}
-                    name="reference"
-                  />
-                  <Text
-                    size="md"
-                    lh="1"
-                    className="ml1 flex align-center text-bold"
-                  >
-                    {t`Learn about our data`}
-                  </Text>
-                </BrowseHeaderIconContainer>
-              </Link>
-            </div>
+            {tab === "databases" && (
+              <div
+                className="flex flex-align-right"
+                style={{ flexBasis: "40.0%" }}
+              >
+                <Link className="flex flex-align-right" to="reference">
```

Comment:
For some reason the link text isn't turning blue on hover when I try this locally, but the icon is.
![Screenshot 2024-02-09 at 9 22 37 AM](https://github.com/metabase/metabase/assets/5248953/54d8a3e0-e3b1-4b8b-9fa0-56ddb0991c61)